athlinks-schema.ttl

```diff
@@ -147,8 +162,47 @@ athlinks:pace
   a owl:DataTypeProperty ;
   rdfs:range schema:DateTime .
 
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:require_dob
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:volunteer
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+For AthleticEvent PROPERTIES
+athlinks:details
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:description
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
```

Comment:
We already have "label" and "comment" on AthleticEvent. Why do we need details and description?